source/includes/_invoice-payment.md

```diff
@@ -258,17 +264,26 @@ KillBillClient::Model::InvoicePayment.refund(payment_id,
 ```python
 invoicePaymentApi = killbill.InvoicePaymentApi()
 
-payment_id = '8d85a8e8-c94b-438f-aac1-e8cb436b2c05'
+paymentId = '8d85a8e8-c94b-438f-aac1-e8cb436b2c05'
 
-transactionBody = killbill.PaymentTransaction(amount=50.0)
+transactionBody = killbill.InvoicePaymentTransaction(amount=50.0)
```

Comment:
I would suggest changing this to `transactionBody = InvoicePaymentTransaction(amount=50.0)` to keep it consistent with other Python code